### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by kunicmarko20**
just approve.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ Closes #{put_issue_number_here}
     Follow this schema: http://keepachangelog.com/
 -->
 
+<!-- 
+    If you are updating something that doesn't require
+    a release, you can delete whole Changelog section.
```
Comment (greg0ire) [inline .github/PULL_REQUEST_TEMPLATE.md]: the whole

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -23,6 +23,12 @@ Closes #{put_issue_number_here}
     Follow this schema: http://keepachangelog.com/
 -->
 
+<!-- 
+    If you are updating something that doesn't require
```
Comment (greg0ire) [inline .github/PULL_REQUEST_TEMPLATE.md]: "does not"

**Comment by greg0ire** [comment]
Rightfully blocked by the `addChild` deprecation issue

**Comment by kunicmarko20** [comment]
Fixed, strange that it comes up just now :thinking:

File: tests/Admin/AdminTest.php [hunk 3]
```diff
@@ -960,7 +960,7 @@ public function testGetBaseCodeRoute(): void
 
         $this->assertSame($postAdmin->getCode(), $postAdmin->getBaseCodeRoute());
 
-        $postAdmin->addChild($commentAdmin);
+        $postAdmin->addChild($commentAdmin, 'post');
```
Comment (greg0ire) [inline tests/Admin/AdminTest.php]: Should probably be backported to 3.x, right?

Comment (kunicmarko20) [reply tests/Admin/AdminTest.php]: Will check but it is strange that there was no error when merging that addChildren PR

Comment (greg0ire) [reply tests/Admin/AdminTest.php]: > Probably because of `$RELEVANT_FILES`

**Comment by kunicmarko20** [comment]
This is green, cc @sonata-project/contributors

File: tests/Twig/Extension/SonataAdminExtensionTest.php [hunk 4]
```diff
@@ -1947,6 +1947,10 @@ public function testGetValueFromFieldDescriptionWithNoValueExceptionNewAdminInst
         );
     }
 
+    /**
+     * @group legacy
+     * @expectedDepreaction The Sonata\AdminBundle\Admin\AbstractAdmin::getTemplate method is deprecated (since 3.34, will be dropped in 4.0. Use TemplateRegistry services instead)
```
Comment (greg0ire) [inline tests/Twig/Extension/SonataAdminExtensionTest.php]: Lol "dep reaction"

**Pull request merged**